websites/P/presence.ts

```diff
@@ -0,0 +1,166 @@
+          details: "Looking the",
+          state: "Homepage",
+          largeImageKey: "logo"
+      };
+      presence.setActivity(homepage);
+  }
+  else if (document.location.pathname.startsWith("/dashboard/")) {
+    const dashboard = {
+        details: "Dashboard",
+        state: document.location.pathname.split("/dashboard/").join(" ").toUpperCase(),
+        largeImageKey: "logo"
+    };
+    presence.setActivity(dashboard);
+}
+  else if (document.location.pathname.includes("posts")) {
+      const posts = {
+          details: "Looking..",
+          state: "Posts page",
+          largeImageKey: "logo"
+      };
+      presence.setActivity(posts);
+  }
+  else if (document.location.pathname.startsWith("/manage/benefits")) {
+      const benefits = {
+          details: "Looking..",
+          state: "Benefits",
+          largeImageKey: "logo"
+      };
+      presence.setActivity(benefits);
+  }
+  else if (document.location.pathname.startsWith("/members")) {
+      const members = {
+          details: "Looking Members",
+          state: "In settings",
+          largeImageKey: "logo"
+      };
+      presence.setActivity(members);
+  }
+  else if (document.location.pathname.includes("members")) {
+    const membersData = {
+        details: "Looking Members..",
+        state: "Benefits",
+        largeImageKey: "logo"
+    };
+    presence.setActivity(membersData);
+}
+  else if (document.location.pathname.startsWith("/creator-home")) {
+    const crhome = {
+        details: "On Homepage..",
+        state: "Creator Home",
+        largeImageKey: "logo"
+    };
+    presence.setActivity(crhome);
+}
+  else if (document.location.pathname.includes("checkout")) {
+    const checkout = {
+      details: "On Checkout..",
+      state: document.querySelector(".gGokLh")?.textContent,
+      largeImageKey: "logo"
+    };
+    presence.setActivity(checkout);
+}
+  else if (document.querySelector(".sc-eCssSg.jjArWB")?.textContent) { //2 view any page
+      const creatorPage = {
+          details: "Viewing a page:",
+          state: document.querySelector(".sc-eCssSg.jjArWB")?.textContent,
+          largeImageKey: "logo"
+    };
+    presence.setActivity(creatorPage);
+}
+else if (document.querySelector(".bHwEqN")?.textContent) { //2 view any page
+  const creatorPage = {
+      details: "Viewing a page:",
+      state: document.querySelector(".bHwEqN")?.textContent,
+      largeImageKey: "logo"
+  };
+  presence.setActivity(creatorPage);
+}
+  else if (document.querySelector(".jAFvqi")?.textContent) { //2 view any page
+  const pagenotfound = {
+      details: "Viewing 404 Error",
+      state: "Page.",
+      largeImageKey: "logo"
+  };
+  presence.setActivity(pagenotfound);
+}
+  else if (document.location.pathname.startsWith("/")) { //
+    const unsupportedPage = {
+        details: "Viewing a",
+        state: "Unsupported page",
+        largeImageKey: "logo"
+  };
+    presence.setActivity(unsupportedPage);
+  }
```

Comment:
Just do 
```
presence.setActivity({
        details: "Viewing a",
        state: "Unsupported page",
        largeImageKey: "logo"
  });
```

Or make an object in the beginning, assign values to that, and then do setActivity once at the end.